### PR TITLE
Rewrite sparse vector index section

### PR DESCRIPTION
**Pull request by mjang**
As discussed internally, we want to improve how we describe Sparse vector indexes. I think (from the PR build) [this](https://deploy-preview-652--condescending-goldwasser-91acf0.netlify.app/documentation/concepts/indexing/#sparse-vector-index) is better.

### CODE REVIEW

File: qdrant-landing/content/documentation/concepts/indexing.md [hunk 1]
```diff
@@ -314,29 +314,20 @@ performance.
...
-### Index Storage and Management
-- **Memory-Based Index:** The index resides in memory for appendable segments, ensuring fast search and update operations.
-- **Handling Immutable Segments:** For immutable segments, the sparse index can either stay in memory or be mapped to disk with the `on_disk` flag.
+To optimize your search, you should reduce the number of vectors that need to be scored. To do this, you need to switch your search from the payload index to the sparse vector index. The `full_scan_threshold` variable configures the switch. It specifies the threshold in terms of the number of matching vectors found by the query planner.
 
-**Example Configuration:** To enable on-disk storage for immutable segments and full scan for queries inspecting less than 5000 vectors:
+This sample configuration enables on-disk storage for immutable segments. It also sets a full scan threshold for queries inspecting less than 5000 vectors.
 
-```http
+```
```
Comment (mjang) [inline qdrant-landing/content/documentation/concepts/indexing.md:330]:
````suggestion
```http
````

File: qdrant-landing/content/documentation/concepts/indexing.md [hunk 2]
```diff
@@ -315,27 +315,18 @@ performance.
...
 
-### Key Features of Sparse Vector Index
-- **Support for Sparse Vectors:** Qdrant supports sparse vectors, characterized by a high proportion of zeroes.
-- **Efficient Indexing:** Utilizes an inverted index structure to store vectors for each non-zero dimension, optimizing memory and search speed.
+Qdrant supports a separate index for Sparse Vectors. Each data point in Qdrant can have both dense and sparse vectors. Therefore, you can still use the same collection for both vector types.  The indexing process is highly efficient, whereby an inverted index structure stores vectors for each non-zero dimension. This approach reduces memory usage and increases search speeds, particularly for high-dimensional data. 
 
-### Search Mechanism
-- **Index Usage:** The index identifies vectors with non-zero values in query dimensions during a search.
-- **Scoring Method:** Vectors are scored using the dot product.
+During search, the index identifies vectors with non-zero values in query dimensions. They are scored using the dot product method. You may also use filtering to search through sparse vectors. Just as with dense vectors, filtering is done by payload.
```
Comment (generall) [inline qdrant-landing/content/documentation/concepts/indexing.md:320]: > During search, the index identifies vectors with non-zero values in query dimensions.

I am not sure this is a best way to describe what happens.

File: qdrant-landing/content/documentation/concepts/indexing.md [hunk 3]
```diff
@@ -315,27 +315,18 @@ performance.
...
-- **`full_scan_threshold` Configuration:** Allows control over when to switch search from the payload index to minimize scoring vectors.
-- **Threshold for Sparse Vectors:** Specifies the threshold in terms of the number of matching vectors found by the query planner.
+The index resides in memory for appendable segments, ensuring fast search and update operations.
+For immutable segments, the sparse index can either stay in memory or be mapped to disk with the `on_disk` flag.
 
-### Index Storage and Management
-- **Memory-Based Index:** The index resides in memory for appendable segments, ensuring fast search and update operations.
-- **Handling Immutable Segments:** For immutable segments, the sparse index can either stay in memory or be mapped to disk with the `on_disk` flag.
+To optimize your search, you should reduce the number of vectors that need to be scored. To do this, you need to switch your search from the payload index to the sparse vector index. The `full_scan_threshold` variable configures the switch. It specifies the threshold in terms of the number of matching vectors found by the query planner.
```
Comment (generall) [inline qdrant-landing/content/documentation/concepts/indexing.md:327]: I don't understand this

Comment (mjang) [reply qdrant-landing/content/documentation/concepts/indexing.md:327]: Trying again:

```suggestion
To optimize your search, switch to the sparse vector index. It is more efficient than the payload index, as it reduces the number of vectors to be scored. You can configure the switch with the `full_scan_threshold` variable, as it specifies the threshold in terms of the number of matching vectors found by the query planner.
```

File: qdrant-landing/content/documentation/concepts/indexing.md (hunk 2 above)
Comment (mjang) [reply qdrant-landing/content/documentation/concepts/indexing.md:320]: We can be more specific. (Is it "the same" or just "similar" query dimensions?)

```suggestion
During search, the index identifies vectors with non-zero values in the same query dimensions. They are scored using the dot product method. You may also use filtering to search through sparse vectors. Just as with dense vectors, filtering is done by payload.
```

**Comment by mjang** [comment]
Hi @generall , I've tried again (after more research). I hope my updates clarify the topic.

**Pull request closed without merging**